data/json/mapgen/house18.json

```diff
@@ -0,0 +1,147 @@
+                { "group": "magazines",   "chance": 90, "x": [16,17], "y": [2,2],   "repeat": [1,5] },
+                { "group": "novels",      "chance": 90, "x": [10,10], "y": [5,5],   "repeat": [1,4] },
+                { "group": "office",      "chance": 80, "x": [17,19], "y": [22,22], "repeat": [1,4] },
+                { "group": "livingroom",  "chance": 90, "x": [2,6],   "y": [3,8],   "repeat": [1,8] },
+                { "group": "dining",      "chance": 85, "x": [17,19], "y": [5,6],   "repeat": [1,4] },
+                { "group": "dining",      "chance": 70, "x": [4,4],   "y": [14,14], "repeat": [1,8] },
+                { "group": "dining",      "chance": 70, "x": [19,19], "y": [14,14], "repeat": [1,4] },
+                { "group": "fridge",      "chance": 90, "x": [5,5],   "y": [14,14], "repeat": [1,6] },
+                { "group": "fridge",      "chance": 90, "x": [18,18], "y": [10,10], "repeat": [1,6] },
+                { "group": "oven",        "chance": 75, "x": [2,2],   "y": [12,12], "repeat": [1,4] },
+                { "group": "oven",        "chance": 75, "x": [20,20], "y": [12,12], "repeat": [1,4] },
+                { "group": "kitchen",     "chance": 90, "x": [2,3],   "y": [10,14], "repeat": [1,4] },
+                { "group": "kitchen",     "chance": 90, "x": [19,20], "y": [10,14], "repeat": [1,4] },
+                { "group": "dresser",     "chance": 85, "x": [2,2],   "y": [18,19], "repeat": [1,9] },
+                { "group": "dresser",     "chance": 85, "x": [2,2],   "y": [22,22], "repeat": [1,5] },
+                { "group": "allclothes",  "chance": 30, "x": [10,10], "y": [11,11], "repeat": [1,6] },
+                { "group": "allclothes",  "chance": 30, "x": [12,12], "y": [11,11], "repeat": [1,6] },
+                { "group": "allclothes",  "chance": 95, "x": [10,10], "y": [22,22], "repeat": [1,5] },
+                { "group": "allclothes",  "chance": 95, "x": [12,12], "y": [22,22], "repeat": [1,5] },
+                { "group": "dresser",     "chance": 90, "x": [20,20], "y": [17,17], "repeat": [1,4] },
+                { "group": "dresser",     "chance": 90, "x": [20,20], "y": [20,20], "repeat": [1,4] },
+                { "group": "softdrugs",   "chance": 75, "x": [8,10],  "y": [19,20], "repeat": [1,5] },
+                { "group": "softdrugs",   "chance": 75, "x": [12,14], "y": [19,20], "repeat": [1,5] },
+                { "group": "harddrugs",   "chance": 15, "x": [8,10],  "y": [19,20], "repeat": [1,2] },
+                { "group": "harddrugs",   "chance": 15, "x": [12,14], "y": [19,20], "repeat": [1,2] },
+                { "group": "cleaning",    "chance": 90, "x": [8,10],  "y": [19,20], "repeat": [1,3] },
+                { "group": "cleaning",    "chance": 90, "x": [12,14], "y": [19,20], "repeat": [1,3] },
+                { "group": "guns_common", "chance": 10, "x": [10,10], "y": [13,13], "ammo": 85, "magazine": 85 },
```

Comment:
The consensus opinion of earlier discussions was that guns in houses should always contain magazines